src/Compilers/Core/Portable/Emit/EmitOptions.cs

```diff
@@ -297,7 +299,8 @@ public bool Equals(EmitOptions? other)
                 IncludePrivateMembers == other.IncludePrivateMembers &&
                 InstrumentationKinds.NullToEmpty().SequenceEqual(other.InstrumentationKinds.NullToEmpty(), (a, b) => a == b) &&
                 DefaultSourceFileEncoding == other.DefaultSourceFileEncoding &&
-                FallbackSourceFileEncoding == other.FallbackSourceFileEncoding;
+                FallbackSourceFileEncoding == other.FallbackSourceFileEncoding &&
+                TestOnly_DataToHexViaXxHash128 == other.TestOnly_DataToHexViaXxHash128;
```

Comment:
>TestOnly_DataToHexViaXxHash128 [](http://example.com/codeflow?start=16&length=30)

Do we need to include this in `Equals()` and `GetHashCode()`? We're not doing that for `_testOnly_AllowLocalStateTracing`.

Reply:
We don't, I thought it might be a safer option, but I can remove it.